sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -32,7 +33,7 @@
 from .._base import build_options as _build_options, validate_cache_staleness_value
 from ..exceptions import CosmosResourceNotFoundError
 from ..http_constants import StatusCodes
-from ..offer import Offer
+from ..offer import ThroughputProperties
```

Comment:
I think we still need to import `Offer`, we just don't need to add it to `__all__`

Reply:
why do we need to import Offer again? It's not being used anywhere anymore with the replacement of ThroughputProperties